cmd/logcli/main.go

```diff
@@ -26,6 +29,9 @@ var (
 )
 
 func main() {
+	if err != nil {
+		// TODO
```

Comment:
Should exit if config file not present or could not be parsed.

Reply:
All done, please look, thanks.

Reply:
seems lint check has some issue.